srcpkgs/ghc/template

```diff
@@ -49,6 +58,14 @@ pre_configure() {
 post_install() {
 	sed -i 's#/usr/lib/ccache/bin/##g' ${DESTDIR}/usr/lib/ghc-${version%[!0-9]}/settings
 	vlicense LICENSE
+
+	if [ "$build_option_bindist" ]; then
+		msg_normal "Creating binary distribution for ${XBPS_TARGET_MACHINE}...\n"
+		make ${makejobs} binary-dist
+		dest="${XBPS_SRCDISTDIR}"/distfiles/ghc-${version}-void-linux-${XBPS_TARGET_LIBC}.tar.xz
+		install -d -m 0644 ghc-${version}-unknown-linux.tar.xz "${dest}"
```

Comment:
this doesn't work, it just creates an empty directory; probably because `ghc-${version}-unknown-linux.tar.xz` is not a thing, there's always an architecture part

also @pullmoll